### ISSUE TITLE
Get similar logits?

### ISSUE THREAD

**Issue by 1073521013** [ThilinaRajapakse/simpletransformers#147]
Excuse me, my task is a multi-classification task. I use this framework to run bert, xlnet, roberta, they are all the same problem, that is, all different inputs have very similar logit, so the prediction results are all Is the same label, but this problem does not occur with bert source, thanks

**Comment by ThilinaRajapakse** [comment]
Is it multiclass or multilabel? Can you show a small example with your code, data, and outputs?

**Comment by acadTags** [comment]
I seem to get similar logits for nearly all instances. I tried the toxic multilabel dataset, and the logit on the dev data is quite strange, while LRAP and eval_loss look good.

```
result: {'LRAP': 0.9948398872003742, 'eval_loss': 0.06017673986752104}
model_outputs: [[0.00515465 0.00017785 0.00134605 0.00020526 0.00122178 0.00033026]
 [0.00475882 0.0001823  0.00132805 0.00020953 0.00120072 0.00032779]
 [0.01551895 0.00020544 0.00189411 0.00029574 0.00220243 0.00056202]
 ...
 [0.00437121 0.00018889 0.00131991 0.00021551 0.00118586 0.0003273 ]
 [0.00413537 0.0001948  0.00132772 0.00022    0.00118117 0.00032845]
 [0.00467253 0.00018354 0.00132504 0.0002107  0.00119689 0.0003275 ]] 
```

**Comment by loganlebanoff** [comment]
I the same problem, with my multi-label classification model getting good metrics on the eval set, but then producing the same logits for every input text. I tried with bert, distilbert, and roberta, and this problem seems to happen only with roberta.

**Comment by loganlebanoff** [comment]
Solved it. I reduced the learning rate from 4e-4 to 3e-5. I don't know why it was still giving good validation loss during training though with the higher learning rate.
